ga4gh/gff3.py

```diff
@@ -0,0 +1,223 @@
+"""
+Object representation of GFF3 data and parser for GFF3 files.
```

Comment:
High level question here: do we need to implement our own GFF parser? Are there no GFF3 Python packages we can leverage? This is a lot of code we'll need to test pretty hard, so it would be nice if someone else had done it already...